src/keys/CompositeKey.cpp

```diff
@@ -64,10 +67,14 @@ CompositeKey& CompositeKey::operator=(const CompositeKey& key)
 
     clear();
 
-    for (const Key* subKey : asConst(key.m_keys)) {
+    Q_FOREACH (const Key* subKey, key.m_keys) {
```

Comment:
This should be a normal for and not a Q_FOREACH, the latter will be deprecated soon.
Is there a reason to roll back to Q_FOREACH?